src/Unity.Mathematics.Shaders/hlsl.cs

```diff
@@ -1,142 +0,0 @@
-using System.Runtime.CompilerServices;
-
-namespace Unity.Mathematics.Shaders
-{
-    public static class hlsl
```

Comment:
Deleting this file appears to be safe since every single release we've done has excluded this from the package.